app/views/Project/Tagging/Sources/SourcesFilter/index.tsx

```diff
@@ -24,61 +24,232 @@ import {
+    SourceFilterOptionsQuery,
+    SourceFilterOptionsQueryVariables,
+} from '#generated/types';
 
+import { getValidDateRangeValues } from '../utils';
+import EntryFilter from './EntryFilter';
 import styles from './styles.css';
 
-// FIXME: Created at and published on are date ranges and not date inputs
+export type SourcesFilterFields = PurgeNull<Pick<EnumFix<ProjectSourcesQueryVariables,
+    'statuses'
+    | 'confidentiality'
+    | 'exists'
+    | 'priorities'
+    | 'statuses'
+>,
+    'statuses'
+    | 'search'
+    | 'exists'
+    | 'priorities'
+    | 'confidentiality'
+    | 'assignees'
+    | 'authoringOrganizationTypes'
+>> & {
+    createdAt?: {
+        startDate: string;
+        endDate: string;
+    }
+    publishedOn?: {
+        startDate: string;
+        endDate: string;
+    }
+    entriesFilterData?: PurgeNull<Pick<EnumFix<LeadEntriesFilterData,
+        'commentStatus'
+        | 'entryTypes'
+    >,
+        'commentStatus'
+        | 'entryTypes'
+    >> & {
+        controlled?: 'true' | 'false';
+        createdBy?: string[];
+        createdAt?: {
+            startDate: string;
+            endDate: string;
+        }
+    }
+};
 
-type FormSchema = ObjectSchema<FilterFormType>;
+type FormType = SourcesFilterFields;
+type FormSchema = ObjectSchema<FormType>;
 type FormSchemaFields = ReturnType<FormSchema['fields']>;
 
 const schema: FormSchema = {
     fields: (): FormSchemaFields => ({
-        status: [],
+        statuses: [],
         createdAt: [],
         publishedOn: [],
-        assignee: [],
+        assignees: [],
         search: [],
         exists: [],
-        priority: [],
+        priorities: [],
         authoringOrganizationTypes: [],
         confidentiality: [],
-        emmRiskFactors: [],
-        emmKeywords: [],
-        emmEntities: [],
+        entriesFilterData: [],
     }),
 };
 
-const initialValue: FilterFormType = {};
+const initialValue: FormType = {};
+
+const SOURCE_FILTER_OPTIONS = gql`
+    query SourceFilterOptions(
+        $projectId: ID!,
+    ) {
+        sourceStatusOptions: __type(name: "LeadStatusEnum") {
+            name
+            enumValues {
+                name
+                description
+            }
+        }
+        sourceExistsOptions: __type(name: "LeadExistsEnum") {
+            name
+            enumValues {
+                name
+                description
+            }
+        }
+        sourcePriorityOptions: __type(name: "LeadPriorityEnum") {
+            name
+            enumValues {
+                name
+                description
+            }
+        }
+        sourceConfidentialityOptions: __type(name: "LeadConfidentialityEnum") {
+            name
+            enumValues {
+                name
+                description
+            }
+        }
+        project(id: $projectId) {
+            members {
+                id
+                displayName
+                firstName
+                lastName
+            }
+        }
```

Comment:
members should not be pulled eagerly. These are later pulled using lazy select inputs